src/gta/enums.hpp

```diff
@@ -2032,4 +2023,467 @@ enum class GameMode : int32_t
+	CPED_CONFIG_FLAG_UseOverrideFootstepPtFx                            = 421,
+	CPED_CONFIG_FLAG_DisableVehicleCombat                               = 422,
+	_CPED_CONFIG_FLAG_0xFE401D26                                        = 423,
+	CPED_CONFIG_FLAG_FallsLikeAircraft                                  = 424,
+	_CPED_CONFIG_FLAG_0x2B42AE82                                        = 425,
+	CPED_CONFIG_FLAG_UseLockpickVehicleEntryAnimations                  = 426,
+	CPED_CONFIG_FLAG_IgnoreInteriorCheckForSprinting                    = 427,
+	CPED_CONFIG_FLAG_SwatHeliSpawnWithinLastSpottedLocation             = 428,
+	CPED_CONFIG_FLAG_DisableStartEngine                                 = 429,
+	CPED_CONFIG_FLAG_IgnoreBeingOnFire                                  = 430,
+	CPED_CONFIG_FLAG_DisableTurretOrRearSeatPreference                  = 431,
+	CPED_CONFIG_FLAG_DisableWantedHelicopterSpawning                    = 432,
+	CPED_CONFIG_FLAG_UseTargetPerceptionForCreatingAimedAtEvents        = 433,
+	CPED_CONFIG_FLAG_DisableHomingMissileLockon                         = 434,
+	CPED_CONFIG_FLAG_ForceIgnoreMaxMeleeActiveSupportCombatants         = 435,
+	CPED_CONFIG_FLAG_StayInDefensiveAreaWhenInVehicle                   = 436,
+	CPED_CONFIG_FLAG_DontShoutTargetPosition                            = 437,
+	CPED_CONFIG_FLAG_DisableHelmetArmor                                 = 438,
+	_CPED_CONFIG_FLAG_0xCB7F3A1E                                        = 439,
+	_CPED_CONFIG_FLAG_0x50178878                                        = 440,
+	CPED_CONFIG_FLAG_PreventVehExitDueToInvalidWeapon                   = 441,
+	CPED_CONFIG_FLAG_IgnoreNetSessionFriendlyFireCheckForAllowDamage    = 442,
+	CPED_CONFIG_FLAG_DontLeaveCombatIfTargetPlayerIsAttackedByPolice    = 443,
+	CPED_CONFIG_FLAG_CheckLockedBeforeWarp                              = 444,
+	CPED_CONFIG_FLAG_DontShuffleInVehicleToMakeRoom                     = 445,
+	CPED_CONFIG_FLAG_GiveWeaponOnGetup                                  = 446,
+	CPED_CONFIG_FLAG_DontHitVehicleWithProjectiles                      = 447,
+	CPED_CONFIG_FLAG_DisableForcedEntryForOpenVehiclesFromTryLockedDoor = 448,
+	CPED_CONFIG_FLAG_FiresDummyRockets                                  = 449,
+	CPED_CONFIG_FLAG_PedIsArresting                                     = 450,
+	CPED_CONFIG_FLAG_IsDecoyPed                                         = 451,
+	CPED_CONFIG_FLAG_HasEstablishedDecoy                                = 452,
+	CPED_CONFIG_FLAG_BlockDispatchedHelicoptersFromLanding              = 453,
+	CPED_CONFIG_FLAG_DontCryForHelpOnStun                               = 454,
+	_CPED_CONFIG_FLAG_0xB68D3EAB                                        = 455,
+	CPED_CONFIG_FLAG_CanBeIncapacitated                                 = 456,
+	_CPED_CONFIG_FLAG_0x4BD5EBAD                                        = 457,
+	CPED_CONFIG_FLAG_DontChangeTargetFromMelee                          = 458,
```

Comment:
We've previously opted to not push these because of their shady origin, please remove these.

Reply:
That sucks. :(